frontend/src/consts/apiConsts.ts

```diff
@@ -0,0 +1 @@
+export const API_URL = 'http://localhost:8080/api/v1/k8s';
```

Comment:
It should be parametrized, but maybe for now it's good

Reply:
This has already been implemented in main. Please resolve the conflicts.